src/components/CarouselCard/indicators/utils.js

```diff
@@ -0,0 +1,23 @@
+export function getChildTabNodes(ref) {
+    if (ref) {
+        return ref.querySelectorAll('[role="tab"]');
+    }
+    return [];
+}
+
+function getRefIndex(nodes, ref) {
+    return nodes.indexOf(ref);
+}
+
+function sortChildren(childrenRefs, nodes) {
+    const [...newChildrenRefs] = childrenRefs;
+    newChildrenRefs.sort(
+        (refA, refB) => getRefIndex(nodes, refA.ref) - getRefIndex(nodes, refB.ref),
+    );
+    return newChildrenRefs;
+}
+
+export function insertChildOrderly(childrenRefs, childRef, nodes) {
+    const newChildrenRefs = childrenRefs.concat([childRef]);
+    return sortChildren(newChildrenRefs, nodes);
+}
```

Comment:
These last three methods are the same that are in the other utils file, then let's remove these and use the ones in the other utils and only keep the first one that is different.